packages/vm/src/compatibility.rs

```diff
@@ -15,6 +15,7 @@ const SUPPORTED_IMPORTS: &[&str] = &[
     "env.db_remove",
     "env.canonicalize_address",
     "env.humanize_address",
+    "env.verify_secp256k1",
```

Comment:
Let's namespace this by algorithm

```suggestion
    "env.secp256k1_verify",
```